### ISSUE TITLE
Mac - Mojave - connection issues and resolution

### ISSUE THREAD

**Issue by demcanulty** [Yepkit/pykush#7]
I'm leaving this note mostly to save people time who may have encountered a similar issue.  I have a YKUSH release 2 (firmware 1.0) purchased in October 2019 that I could not seem to talk to.  ykushcmd compiled fine but could not open the device (this is expected as mac compability is not claimed).  

Running pykush from code did not seem to work, it would fail by exceeding the recursion limit.  If I tried to code it directly using hidapi I would get "File "hid.pyx", line 66, in hid.device.open OSError: open failed."   

Running as root didn't seem to fix any of the above issues.  

YKUSH would not successfully connect to my windows 10 vm ("Device failed to connect because of driver error").

I could not compile ykushcmd on my windows 10 laptop because it said files were missing and I wasn't able to track down what was broken about the repository or if there was something wrong about my build chains, but I got the same failure on two different machines with fresh installations of MinGW.  

Finally! I realized that there were precompiled binaries for pykush, and those worked fine on my windows 10 laptop and I was able to control the device.  I subsequently determined that the pykush binary also works on my mac, and that one issue had been that the YKUSH was plugged into a series of daisy chained usb hubs (#donglelife #thestruggleisreal) and that something about the final usb hub and the YKUSH did not agree.  

ykushcmd, running pykush from code, and manually connecting with hidapi still doesn't work on my mac.  But the pykush binary works reliably now in all situations, including my windows 10 vmware.  

So, to summarize, if anybody else is struggling just to get basic connectivity to the device, I recommend using the pykush binaries, and removing any intervening usb hubs between your computer and the YKUSH.
